cmd/klepto.go

```diff
@@ -35,12 +35,12 @@ func init() {
 	Klepto.PersistentFlags().StringVarP(&configFile, "config", "c", "", "Path to config file (default is $HOME/.klepto.toml)")
 	Klepto.PersistentFlags().StringVarP(&fromDSN, "from", "f", "root:root@tcp(localhost:3306)/klepto", "MySQL database dsn to steal from")
 	Klepto.PersistentFlags().StringVarP(&toDSN, "to", "t", "", "MySQL database to output to (default writes to stdOut)")
-	Klepto.PersistentFlags().StringVarP(&pRecordType, "primary-record-type", "p", "", "Name of specific table or record type you want to steal")
+	Klepto.PersistentFlags().StringVarP(&pRecordType, "primary_record_type", "p", "", "Name of specific table or record type you want to steal")
```

Comment:
Not sure about having underscores in the flag

Reply:
Yeah, you're right. Looks a bit awkward. Did this to keep the key holding the config consistent with the toml configuration as well. That way, the same key holds the config and viper will choose one (I think command line) over the other if both were set.

Reply:
Probably we don't need the CLI flag override in this case, then

Reply:
Removed it, then. Simpler. Me like :)